model/version_activation.go

```diff
@@ -120,7 +120,7 @@ func ActivateElapsedBuildsAndTasks(v *Version) (bool, error) {
 	}
 	if len(buildIdsToActivate) > 0 {
 		// Don't need to set the version in here since we do it ourselves in a single update
-		if err := build.UpdateActivation(buildIdsToActivate, true, evergreen.DefaultTaskActivator); err != nil {
+		if err := build.UpdateActivation(buildIdsToActivate, true, evergreen.BuildActivator); err != nil {
```

Comment:
I'm actually maybe confused about the difference between the activation here and below but is it confusing to have two different activator names here?

Reply:
Oh it seems like we activate the build if either the whole build or individual tasks are being activated. Maybe for traceability it's fine to leave this as is.